ml_ops.sh

```diff
@@ -74,13 +74,13 @@ hdfs dfs -rm -R -f ${HDFS_SCORED_CONNECTS}
 
 # Add -p <command> to execute pre MPI command.
 # Pre MPI command can be configured in /etc/duxbay.conf
-# In this script, after the line after --mpicmd ${MPI_CMD} add:
-# --mpiprep ${MPI_PREP_CMD}
+# In this script, after the line after -c ${MPI_CMD} add:
+# -p ${MPI_PREP_CMD}
 
 
-time spark-submit --class "org.opennetworkinsight.LDA" --master yarn-client --executor-memory  ${SPK_EXEC_MEM} \
+time spark-submit --class "org.opennetworkinsight.SuspiciousConnects" --master yarn-client --executor-memory  ${SPK_EXEC_MEM} \
```

Comment:
I like this name... we should always try to use names that match WHAT the function does not HOW it does it... in this particular case, LDA is just one step in a larger "suspicious connects" analysis